nuxt.config.js

```diff
@@ -98,7 +98,7 @@ export default {
     { src: '~/plugins/polkadot', mode: 'client' },
     { src: '~/plugins/endpoint', mode: 'client' },
     { src: '~/plugins/seoMetaGenerator', mode: 'client' },
-    { src: '~/plugins/keyboardEvents', mode: 'client' },
+    { src: '~/plugins/appLoad', mode: 'client' },
```

Comment:
typo?

Reply:
No @vikiival I have renamed the file name to `appLoad`, so that we can write code which needs to be run on app first load, like binding keyboard events and fetching user balance.

Reply:
Yeah, use separated file for that